nix/byron-proxy-scripts.nix

```diff
@@ -0,0 +1,26 @@
+{ commonLib, oldCardanoSrc, byronProxy }:
+
+let
+  pkgs = commonLib.pkgs;
+  oldCardano = import oldCardanoSrc {};
+  oldCardanoLib = import (oldCardanoSrc + "/lib.nix");
+  environments = oldCardanoLib.environments;
+  loggingConfig = ../byron-proxy/cfg/logging.yaml;
+  mkTopology = relay: pkgs.writeText "topology-file" ''
+    wallet:
+      relays: [[{ host: ${relay} }]]
+      valency: 1
+      fallbacks: 7
```

Comment:
These `valency` and `fallbacks` keys actually aren't necessary: they are implicit in the dimensions of the `relays` list. Valency is the outer length, fallbacks for each entry is its length.